src/components/change-requests/new-change-request/new-change-request-page/standard-form-fields/standard-form-fields.tsx

```diff
@@ -7,20 +7,43 @@ import { Row, Col, Form, InputGroup, FormControl } from 'react-bootstrap';
 import { exampleAllWorkPackages } from '../../../../../test-support/test-data/work-packages.stub';
 import { wbsPipe } from '../../../../../shared/pipes';
 import styles from './standard-form-fields.module.css';
+import { Scope_CR_Why_Type } from '@prisma/client';
 
-const StandardFormFields: React.FC = () => {
+interface IProp {
+  handleChange: (e: any) => void,
+}
+
+const whyInputName = {
```

Comment:
definitely a tech debt ticket here?

Reply:
at the very least for swapping to not use prisma types? https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/dca5d7b78d63c608c9cac301fdd9b27361f35cd4/src/utils/src/types/change-request-types.ts#L56